packages/gatsby/src/state-machines/develop/actions.ts

```diff
@@ -15,6 +15,7 @@ import reporter from "gatsby-cli/lib/reporter"
 import { ProgramStatus } from "../../redux/types"
 import { createWebpackWatcher } from "../../services/listen-to-webpack"
 import { callRealApi } from "../../utils/call-deferred-api"
+
```

Comment:
Please drop (if a file has no change it shouldn't be marked as changed by this PR)